test/testcafe/spec/EnrollAuthenticatorPasswordView_spec.js

```diff
@@ -98,8 +98,8 @@ test.requestHooks(successMock)('should have the correct reqiurements', async t =
   await t.expect(enrollPasswordPage.getRequirements()).contains('A symbol');
   await t.expect(enrollPasswordPage.getRequirements()).contains('Does not include your first name');
   await t.expect(enrollPasswordPage.getRequirements()).contains('Does not include your last name');
-  await t.expect(enrollPasswordPage.getRequirements()).contains('At least 2 hour(s) must have elapsed since you last changed your password');
+  // await t.expect(enrollPasswordPage.getRequirements()).contains('At least 2 hour(s) must have elapsed since you last changed your password');
```

Comment:
These 2 messages aren't available in the v3 widget for password authenticator enrollment page.
I'm not sure if these messages are from server through the response or rendered by the widget.

Reply:
In v3, UX & Product actually made a decision to _not_ display the server side messages because they can be sort of confusing. There is another ticket pending final work from UX that will re-introduce these server side messages but in a different context.

For now, I would actually recommend wrapping this in a condition (to not impact v2 tests but exclude it for v3), this is one of the rare times that we include conditional statements in the test, you can mimic what we did in this test: https://github.com/okta/okta-signin-widget/blob/0.1/test/testcafe/spec/ReEnrollAuthenticatorWarningPasswordView_spec.js#L84-L89

Reply:
Do we have a ticket tracking the re-introduction of this message with new UX? If so, we should include it in the comment explaining why we are skipping this assertion in v3.

Reply:
Yep! We have that ticket documented in the other test too. `OKTA-533383`

If you want to copy the exact comment that the link above has I think that is totally acceptable since they are pretty much equivalent.